src/app/shared/object-list/my-dspace-result-list-element/item-list-preview/item-list-preview.component.html

```diff
@@ -44,7 +44,7 @@ <h2 [innerHTML]="dsoTitle" [ngClass]="{'lead': true,'text-muted': !item.firstMet
             </ds-truncatable-part>
           </span>
           <ds-truncatable-part [id]="item.id" [minLines]="1" class="item-list-abstract">
-            <span [ngClass]="{'text-muted': !item.firstMetadataValue('dc.description.abstract')}"
+            <span [ngClass]="{'text-muted': !item.hasMetadata('dc.description.abstract')}"
             [innerHTML]="(item.firstMetadataValue('dc.description.abstract')) || ('mydspace.results.no-abstract' | translate)"></span>
```

Comment:
This line is not correctly updated based on #4737 either.  It should use: `item.firstMetadataValue('dc.description.abstract', undefined, true)`